x-pack/plugins/lists/common/schemas/request/create_exception_list_item_schema.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable @typescript-eslint/camelcase */
+
+import * as t from 'io-ts';
+
+import {
+  ItemId,
+  Tags,
+  _Tags,
+  _tags,
+  comment,
+  description,
+  exceptionListItemType,
+  list_id,
+  meta,
+  name,
+  tags,
+} from '../common/schemas';
+import { Identity, RequiredKeepUndefined } from '../../types';
+import { DefaultEntryArray, DefaultUuid } from '../types';
+import { EntriesArray } from '../types/entries';
+
+export const createExceptionListItemSchema = t.intersection([
+  t.exact(
+    t.type({
+      description,
+      list_id,
+      name,
+      type: exceptionListItemType,
+    })
+  ),
+  t.exact(
+    t.partial({
+      _tags, // defaults to empty array if not set during decode
+      comment, // defaults to undefined if not set during decode
```

Comment:
Would we want to default to array? Regardless of the comment structure, this will either be a list of ids or list of objects?

Reply:
Oh yeah, hehe... This should be an array, you are correct. I will change that really quickly. Appreciate the sharp 👁️ 's on this one. I will make it to an array and default it.